generator broken: error TS18003: No inputs were found in config file

Generator fails upon running this generator with default settings. 

npm install -g yo generator-theia-extension
mkdir theia-hello-world-extension
cd theia-hello-world-extension
yo theia-extension hello-world-extension

```
hello-world-extension: $ tsc
hello-world-extension: error TS18003: No inputs were found in config file '/home/ubuntu/environment/theia-hello-world-extension/hello-world-extension/tsconfig.json'. Specified 'include' paths were '["src"]' and 'exclude' paths were '["lib"]'.
hello-world-extension: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
hello-world-extension: error Command failed with exit code 2.
hello-world-extension: npm ERR! code ELIFECYCLE
hello-world-extension: npm ERR! errno 2
hello-world-extension: npm ERR! hello-world-extension@0.0.0 prepare: `yarn run clean && yarn run build`
hello-world-extension: npm ERR! Exit status 2
hello-world-extension: npm ERR! 
hello-world-extension: npm ERR! Failed at the hello-world-extension@0.0.0 prepare script.
hello-world-extension: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
hello-world-extension: npm ERR! A complete log of this run can be found in:
hello-world-extension: npm ERR!     /home/ubuntu/.npm/_logs/2019-08-19T15_54_36_233Z-debug.log
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm run prepare
lerna ERR! warning package.json: No license field
lerna ERR! warning package.json: No license field
lerna ERR! error Command failed with exit code 2.
lerna ERR! npm ERR! code ELIFECYCLE
lerna ERR! npm ERR! errno 2
lerna ERR! npm ERR! hello-world-extension@0.0.0 prepare: `yarn run clean && yarn run build`
lerna ERR! npm ERR! Exit status 2
lerna ERR! npm ERR! 
lerna ERR! npm ERR! Failed at the hello-world-extension@0.0.0 prepare script.
lerna ERR! npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
lerna ERR! 
lerna ERR! npm ERR! A complete log of this run can be found in:
lerna ERR! npm ERR!     /home/ubuntu/.npm/_logs/2019-08-19T15_54_36_233Z-debug.log
lerna ERR! 
lerna ERR! 
lerna ERR! > hello-world-extension@0.0.0 prepare /home/ubuntu/environment/theia-hello-world-extension/hello-world-extension
lerna ERR! > yarn run clean && yarn run build
lerna ERR! 
lerna ERR! yarn run v1.17.3
lerna ERR! $ rimraf lib
lerna ERR! Done in 0.17s.
lerna ERR! yarn run v1.17.3
lerna ERR! $ tsc
lerna ERR! error TS18003: No inputs were found in config file '/home/ubuntu/environment/theia-hello-world-extension/hello-world-extension/tsconfig.json'. Specified 'include' paths were '["src"]' and 'exclude' paths were '["lib"]'.
lerna ERR! info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! 
lerna ERR!     at Promise.all.then.arr (/home/ubuntu/environment/theia-hello-world-extension/node_modules/execa/index.js:236:11)
(node:16576) UnhandledPromiseRejectionWarning: Error: Command failed: npm run prepare
warning package.json: No license field
warning package.json: No license field
error Command failed with exit code 2.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! hello-world-extension@0.0.0 prepare: `yarn run clean && yarn run build`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the hello-world-extension@0.0.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2019-08-19T15_54_36_233Z-debug.log
```

> hello-world-extension@0.0.0 prepare /home/ubuntu/environment/theia-hello-world-extension/hello-world-extension
> yarn run clean && yarn run build

yarn run v1.17.3
$ rimraf lib
Done in 0.17s.
yarn run v1.17.3
$ tsc
error TS18003: No inputs were found in config file '/home/ubuntu/environment/theia-hello-world-extension/hello-world-extension/tsconfig.json'. Specified 'include' paths were '["src"]' and 'exclude' paths were '["lib"]'.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

    at Promise.all.then.arr (/home/ubuntu/environment/theia-hello-world-extension/node_modules/execa/index.js:236:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:16576) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:16576) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
`

--- a/src/app/index.ts
+++ b/src/app/index.ts
@@ -127,7 +127,10 @@ module.exports = class TheiaExtension extends Base {
             extensionName;
         const extensionPath = path.normalize(unscopedExtensionName).replace('/', '-');
         const extensionPrefix = extensionPath.split('-').map(name => this._capitalize(name)).join('');
-        const extensionType = options.extensionType as number;
+        let extensionType = options.extensionType as number;
+        if(extensionType === undefined) {
+            extensionType = 1;
+        }
         this.log(extensionPrefix)
         this.params = {
             ...options,

